modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml

```diff
@@ -54,7 +54,11 @@
             <class name="org.wso2.am.integration.tests.other.APIMANAGER3226APINameWithDifferentCaseTestCase"/>
         </classes>
     </test>
-
+    <test name="apim-integration-tests-token" preserve-order="true" parallel="false">
```

Comment:
Any reason to add this as a separate test block?

Reply:
since this is under the package "org.wso2.am.integration.tests.token"